treeherder/webapp/api/logslice.py

```diff
@@ -59,10 +59,6 @@ def list(self, request, project, jm):
         if start_line >= end_line:
             return Response("``end_line`` must be larger than ``start_line``", 400)
 
-        # @todo: remove once Bug 1139517 is addressed
-        if log_name == "buildbot_text":
-            log_name = 'builds-4h'
-
         # get only the log that matches the ``log_name``
         logs = jm.get_log_references(job_id)
 
```

Comment:
in the following ``log = next(log for log in logs if log["name"] == log_name)`` it should be changed to something like:

    # @todo: remove after no more logs named 'builds-4h' exist in the db.  Should be after Aug 30, 2015.
    log = next(log for log in logs if log["name"] in [log_name, 'builds-4h'])